LanguageFeatures/primary-constructors/static_processing_A06_t02.dart

```diff
@@ -29,56 +29,64 @@
 
 import '../../Utils/expect.dart';
 
-extension type ET1(String x) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
+String log = "";
 
-  this : captureInInitializer = (() => x);
+extension type ET1(String x) {
+  this : assert("$log=$x".isNotEmpty);
 }
 
 extension type ET2([String x = "default"]) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
-
-  this : captureInInitializer = (() => x);
+  this : assert("$log=$x".isNotEmpty);
```

Comment:
Admittedly, we're testing something that looks like nothing, but I do think it makes sense to have a `captureInBody`-like behavior, based on something like `bodyLog = x;` in a constructor body here, and checks on `bodyLog` like the checks on `log` in `main`.